Python/sysmodule.c

```diff
@@ -2580,6 +2581,7 @@ set_flags_from_config(PyInterpreterState *interp, PyObject *flags)
     SetFlagObj(PyBool_FromLong(config->dev_mode));
     SetFlag(preconfig->utf8_mode);
     SetFlag(config->warn_default_encoding);
+    SetFlagObj(PyBool_FromLong(config->no_debug_ranges));
```

Comment:
Hummm, I wonder if we should make this private for the time being

Reply:
Not having this here would make it a bit of a pain to test. We'd have to use something like `_testinternalcapi.get_config()`

Reply:
Yes, it also polutes the sys namespace. It either needs to be made private (underscore) or we need to check the config through testcapi

Reply:
Removed from `sys.flags`, the tests now use `_testinternalcapi`.